guides/advanced_guides/configuration.md

```diff
@@ -269,19 +273,17 @@ This command will stop the process if:
 
 If this command is not called, no snapshot will be imported.
 
-Uncompatible with `--recover-from-snapshot`
 
-### Recover from snapshot
+### Ignore missing snapshot
 
-**CLI option**: `--recover-from-snapshot`
+**Environment variable**: `MEILI_IGNORE_MISSING_SNAPSHOT`
+**CLI option**: `--ignore-missing-snapshot`
 
-The path of the snapshot file to import.
-
-This command will skip importing a snapshot and continue the process if:
+The engine will ignore a missing snapshot and not return an error in such case.
```

Comment:
```suggestion
The engine ignores missing snapshots and does not throw an error in this case.
```
What do you mean by that? 
If `--load-from-snapshot` is enabled but their are no snapshots in the `--snapshot-path` then their will be no errors thrown and no snapshots will be loaded? Is this correct?